src/commands/push.js

```diff
@@ -156,8 +156,12 @@ class PushCommand extends ApifyCommand {
 
         if (build.status === ACT_JOB_STATUSES.SUCCEEDED) {
             outputs.success('Actor was deployed to Apify cloud and built there.');
-        } else if (build.status === ACT_JOB_STATUSES.RUNNING) {
+        } else if (build.status === ACT_JOB_STATUSES.RUNNING || build.status === ACT_JOB_STATUSES.READY) {
             outputs.warning('Build is still running!');
```

Comment:
Wouldn't it make sense to be honest about it and use "Build is waiting for allocation to a worker" for the READY state? Because sometimes the ready state takes 5+ minutes to clear and users could think that the build itself was taking this long.

Reply:
I was just talking about splitting the condition into two `else if`s and having different messages. It's definitely not worth changing any backend logic. 👍

Reply:
OK, I'll put "Build is waiting for execution" there